src/components/StatsPanel/StatsPanel.constants.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  WorkOutline,
+  CheckCircleOutline,
+  ThumbDownOffAlt,
+  HourglassEmpty,
+} from '@mui/icons-material';
+
+import type { StatItemData } from './StatsPanel.types';
+
+export const TEST_STATS: Record<string, StatItemData> = {
+  total: { id: 'total', label: 'No of Tests', value: 32, icon: WorkOutline, color: 'primary' },
```

Comment:
Copy/label text 'No of Tests' is grammatically awkward and inconsistent with other labels. Use consistent and clear English labels across the UI.

**Sugerencia:**
```typescript
Change the label to 'Total tests' or 'Number of tests', e.g.:

total: { id: 'total', label: 'Total tests', value: 32, icon: WorkOutline, color: 'primary' },
```